supplements/tashas-cauldron-of-everything/artificer-infusions.xml

```diff
@@ -246,7 +248,436 @@
+		</description>
+		<sheet>
+			<description>You can infuse a simple or martial weapon with the thrown property into a returning weapon.</description>
+			<!-- <description>Item: A simple or martial weapon with the thrown property
+			This magic weapon grants a +1 bonus to attack and damage rolls made with it, and it returns to the wielder’s hand immediately after it is used to make a ranged attack.</description> -->
+		</sheet>
+	</element>
 
+	<element name="Replicate Magic Item Table" type="Information" source="Tasha’s Cauldron of Everything" id="ID_WOTC_TCOE_CLASS_FEATURE_ARTIFICER_INFUSION_REPLICATE_MAGIC_ITEM_TABLES">
+		<compendium display="false" />
+		<description>
+			<p><strong>Replicable Items (2nd-level Artificer)</strong></p>
+			<table>
+				<thead>
+					<tr><td>Magic Item</td><td class="col-20">Attunement</td></tr>
+				</thead>
+				<tr><td>Alchemy jug</td><td>No</td></tr>
+				<tr><td>Bag of holding</td><td>No</td></tr>
+				<tr><td>Cap of water breathing</td><td>No</td></tr>
+				<tr><td>Goggles of night</td><td>No</td></tr>
+				<tr><td>Rope of climbing</td><td>No</td></tr>
+				<tr><td>Sending stones</td><td>No</td></tr>
+				<tr><td>Wand of magic detection</td><td>No</td></tr>
+				<tr><td>Wand of secrets</td><td>No</td></tr>
+			</table>
+			<p><strong>Replicable Items (6th-level Artificer)</strong></p>
+			<table>
+				<thead>
+					<tr><td>Magic Item</td><td class="col-20">Attunement</td></tr>
+				</thead>
+				<tr><td>Boots of elvenkind</td><td>No</td></tr>
+				<tr><td>Cloak of elvenkind</td><td>Yes</td></tr>
+				<tr><td>Cloak of the manta ray</td><td>No</td></tr>
+				<tr><td>Eyes of charming</td><td>Yes</td></tr>
+				<tr><td>Gloves of thievery</td><td>No</td></tr>
+				<tr><td>Lantern of revealing</td><td>No</td></tr>
+				<tr><td>Pipes of haunting</td><td>No</td></tr>
+				<tr><td>Ring of water walking</td><td>No</td></tr>
+			</table>
+			<p><strong>Replicable Items (10th-level Artificer)</strong></p>
+			<table>
+				<thead>
+					<tr><td>Magic Item</td><td class="col-20">Attunement</td></tr>
+				</thead>
+				<tr><td>Boots of striding and springing</td><td>Yes</td></tr>
+				<tr><td>Boots of the winterlands</td><td>Yes</td></tr>
+				<tr><td>Bracers of archery</td><td>Yes</td></tr>
+				<tr><td>Brooch of shielding</td><td>Yes</td></tr>
+				<tr><td>Cloak of protection</td><td>Yes</td></tr>
+				<tr><td>Eyes of the eagle</td><td>Yes</td></tr>
+				<tr><td>Gauntlets of ogre power</td><td>Yes</td></tr>
+				<tr><td>Gloves of missile snaring</td><td>Yes</td></tr>
+				<tr><td>Gloves of swimming and climbing</td><td>Yes</td></tr>
+				<tr><td>Hat of disguise</td><td>Yes</td></tr>
+				<tr><td>Headband of intellect</td><td>Yes</td></tr>
+				<tr><td>Helm of telepathy</td><td>Yes</td></tr>
+				<tr><td>Medallion of thoughts</td><td>Yes</td></tr>
+				<tr><td>Necklace of adaptation</td><td>Yes</td></tr>
+				<tr><td>Peria pt of wound closure</td><td>Yes</td></tr>
```

Comment:
noticed by the user [Alby1987](https://github.com/Alby1987). Konard, please check that typo in other files too
```suggestion
				<tr><td>Periapt of wound closure</td><td>Yes</td></tr>
```